backend/company-agent/src/main/java/org/hyperledger/oa/impl/aries/AriesStartupTasks.java

```diff
@@ -73,46 +78,26 @@ public void onServiceStartedEvent() {
             vpMgmt.recreateVerifiablePresentation();
         });
 
-        credLookup.ifPresent(lookup -> lookup.lookupTypesForAllPartnersAsync());
+        // register endpoints if not in webmode or if no TTA acceptance is required
+        if (webOnly != null && webOnly.booleanValue()) {
```

Comment:
This I do not get. This does the opposite of the comment above, and tries to register an endpoint when in web mode, but as we can never be in web mode here this is dead code anyway.

Reply:
comment is wrong, should be " if not in webmode"